library/objective-c/EnvoyEngine.h

```diff
@@ -326,12 +330,16 @@ extern const int kEnvoyFailure;
  @param logLevel The log level to use when starting Envoy.
  @param onEngineRunning Closure called when the engine finishes its async startup and begins
  running.
+ @param onEngineLog Closure to be called when the engine logs.
+ @param onEngineFlush Closure to be called when the engine's logger flushes.
```

Comment:
I think it could be useful to provide more information here as to when it flushes (and what the difference between a log and flush is in this context)